pandas/core/generic.py

```diff
@@ -8024,6 +8027,21 @@ def resample(
 
             .. versionadded:: 1.1.0
 
+            - 'end': `origin` is the last value of the timeseries
+            - 'end_day': `origin` is the ceiling midnight of the last day
+
+            .. versionadded:: 1.2.0
+
+        backward : bool, default is None
```

Comment:
if anything this should be a separate issue / PR discussion. I actually am -1 on adding this as yet another keyword.

pls revert everything in this PR and focus *only* on adding the end/end_day), though then do these become useless?

Reply:
Sorry for adding without discussion. I will make the reversion in this pr. Maybe the backward resampling on an arbitrary Timestamp needs a new strategy instead of new keyword. (origin=('20201129', 'B') seems to be a possible solution, but a little strange to take this tuple input?)